pkg/asset/installconfig/openstack/validation/platform_test.go

```diff
@@ -64,6 +86,20 @@ func TestOpenStackPlatformValidation(t *testing.T) {
 			expectedError:  false,
 			expectedErrMsg: "",
 		},
+		{
+			// if platform flavor is for both compute and ctrl plane, then it needs to be valid for
+			// the ctrl plane
+			name: "invalid platform flavor",
+			platform: func() *openstack.Platform {
+				p := validPlatform()
+				p.FlavorName = validComputeFlavor
```

Comment:
I suggest you instead use `invalidCtrlPlaneFlavor`, where you can set all the values to be lower than required. You can also drop `validComputeFlavor` and `invalidComputeFlavor` that are not used for platform flavors.

Reply:
My thinking here was that if they opt to use a default, then it has to meet the minimum requirement for both the master and workers. And since the master has higher requirements, then that is the bound we should test for.